src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -697,10 +717,10 @@ func (repo *MarketplaceCmdRepo) UninstallItem(
 		return err
 	}
 
-	installedItemModel := dbModel.MarketplaceInstalledItem{
+	installedServiceItemModel := dbModel.MarketplaceInstalledItem{
 		ID: uint(deleteDto.InstalledId.Uint16()),
```

Comment:
Why MarketplaceInstalledItem ID is an uint but the VO is an uint16? Shouldn't both be an uint64?

Reply:
So, the new ID standard we had started using was `uint16`. I updated the model to match the same standard as the VO.